osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -457,6 +466,18 @@ namespace renderkit {
         delete m_library.OpenGL;
     }
 
+    bool RenderManagerOpenGL::GetTimingInfo(size_t whichEye, RenderTimingInfo& info) {
+      if(m_toolkit.getRenderTimingInfo) {
+        OSVR_RenderTimingInfo renderTimingInfo;
+        bool ret = m_toolkit.getRenderTimingInfo(m_toolkit.data, 0, whichEye, &renderTimingInfo);
```

Comment:
[Style] I'd declare `ret` to be a `const bool` since you're not changing it after assignment. Again, not really a problem here, just a good habit to be in.